### PR TITLE
deploy s3watcher w/teamcity + riffraff

### PR DESCRIPTION
**Pull request by akash1810**

### CODE REVIEW

File: s3watcher/build.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+get_function_name() {
+    echo $(aws cloudformation list-stack-resources --stack-name $1 \
```
Comment (theefer) [inline s3watcher/build.sh]: Let's remember to always pass in the `--profile` explicitly with every `aws` command?

File: s3watcher/build.sh [hunk 2]
```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
```
Comment (theefer) [inline s3watcher/build.sh]: You may want a `set -e` here to abort if anything fails?

File: s3watcher/build.sh (hunk 1 above)
Comment (theefer) [inline s3watcher/build.sh]: Also is this meant to run on TC? How will it have the right credentials to do so?

**Comment by theefer** [comment]
Nice, just a few questions above on the build script.

File: s3watcher/build.sh (hunk 1 above)
Comment (akash1810) [inline s3watcher/build.sh]: > How will it have the right credentials to do so?

Environment variables.

Comment (theefer) [inline s3watcher/build.sh]: Can you have a chat with @adamnfish about whether this is OK security-wise?

**Comment by akash1810** [comment]
merging so I can setup teamcity

**Pull request merged**